pkg/crc/preflight/preflight_linux.go

```diff
@@ -368,6 +368,7 @@ func getChecks(distro *linux.OsRelease, bundlePath string, preset crcpreset.Pres
 	checks = append(checks, genericPreflightChecks(preset)...)
 	checks = append(checks, memoryCheck(preset))
 	checks = append(checks, gvproxyCheck())
+	checks = append(checks, macadamCheck())
```

Comment:
It’s imo a bit odd to have separate `gvproxyCheck()`, `macadamCheck()`, … This could go in one of the existing lists of checks